tensorflow/compiler/xla/tests/llvm_compiler_test.cc

```diff
@@ -147,7 +155,11 @@ TEST_F(CpuCompilerTest, HooksTest) {
 }
 
 TEST_F(GpuCompilerTest, HooksTest) {
+#if GOOGLE_CUDA
```

Comment:
#if !TENSORFLOW_USE_ROCM . let's try avoid GOOGLE_CUDA within XLA, as there really is no CUDA within XLA but only NVPTX.